controllers/capabilities/application_status_reconciler_test.go

```diff
@@ -326,26 +319,6 @@ func getApplicationDeveloperAccount() (CR *capabilitiesv1beta1.DeveloperAccount)
 	return CR
 }
 
-func getApplicationBaseReconciler() (baseReconciler *reconcilers.BaseReconciler) {
-	// Register operator types with the runtime scheme.
-	s := scheme.Scheme
-	s.AddKnownTypes(appsv1alpha1.GroupVersion, getApplicationCR())
-	s.AddKnownTypes(appsv1alpha1.GroupVersion, getApplicationProductCR())
-	s.AddKnownTypes(appsv1alpha1.GroupVersion, getApiManger())
-	s.AddKnownTypes(appsv1alpha1.GroupVersion, getApplicationDeveloperAccount())
-	log := logf.Log.WithName("Application status reconciler test")
-	ctx := context.TODO()
-	// Objects to track in the fake client.
-	objs := []runtime.Object{getApplicationCR(), getProviderAccount(), getApiManger(), getApplicationProductList(), getApplicationDeveloperAccount(), getProviderAccountRefSecret()}
```

Comment:
This was deleted however, I can restore with refactor of the getBaseReconciler func but wanted to make it in a seperate PR that refactors the unit tests.

Reply:
hmm yea I was wondering why there were changes to the application. I will take another look in the morning.